bindings/experimental/distrdf/python/DistRDF/Ranges.py

```diff
@@ -266,8 +264,10 @@ def get_clustered_ranges(clustersinfiles, npartitions, treename, friend_info):
                     cluster.filetuple for cluster in clusters
                 ]), key=lambda curtuple: curtuple[1])
             ],  # type: list[str]
-            friend_info.friend_names,  # type: list[str]
-            friend_info.friend_file_names  # type: list[str]
+            friendnamesalias,  # type: list[str]
+            friendfilenames,  # type: list[str]
+            friendchainsubnames, # type: list[str]
+            defaultbranches,  # type: list[str]
         )  # type: collections.namedtuple
         for clusters in _n_even_chunks(clustersinfiles, npartitions)
     ]
```

Comment:
not for this PR, but this is way too terse, I have no idea what's going on here :sweat_smile:

Reply:
Ok, we'll see what we can do about that in another PR